dockerfiles/dev-env/Dockerfile

```diff
@@ -143,23 +151,24 @@ ENV PETSC_PREFIX /usr/local/petsc
 RUN export PATH=$OPENMPI_PREFIX/bin:$PATH && \
     export LD_LIBRARY_PATH=$OPENMPI_PREFIX/lib:$LD_LIBRARY_PATH && \
     export INCLUDE=$OPENMPI_PREFIX/include:$INCLUDE && \
+    rm -rf /tmp/* && \
+    wget https://github.com/hypre-space/hypre/archive/v2.15.1.tar.gz && \
     git clone -b v3.11.3 -q https://gitlab.com/petsc/petsc.git && \
     cd petsc && \
-    python3 ./configure --COPTFLAGS="-O2" \
-                        --CXXOPTFLAGS="-O2" \
-                        --FOPTFLAGS="-O2" \
-                        --with-clanguage=C++ \
+    python3 ./configure --COPTFLAGS="-O3" \
+                        --CXXOPTFLAGS="-O3" \
+                        --FOPTFLAGS="-O3" \
                         --with-shared-libraries=1 \
+                        --with-openblas=1 \
+                        --with-debugging=0 \
                         --download-suitesparse=1 \
-                        --download-hypre=1 \
+                        --download-hypre=/tmp/v2.15.1.tar.gz \
                         --download-ml=1 \
                         --download-parmetis=1 \
                         --download-metis=1 \
                         --download-blacs=1 \
                         --download-scalapack=1 \
                         --download-mumps=1 \
-                        --download-petsc4py=1 \
-                        --with-debugging=0 \
```

Comment:
--with-debugging just got moved up in the list (or should have)

and petsc4py wouldn't build under Python3.8 (and didn't seem to be needed for the rest of the stack) so I just got lazy and took it out of the petsc build

Reply:
OK, since this is going to live as a separate tag it's not important.  Petsc4py is sometimes useful to have but it's fine if it's removed for now.

Apologies hadn't spotted the debugging move.